### ISSUE TITLE
Wait for data to be available before updating a feed

### ISSUE THREAD

**Issue by rampall** [ethersphere/bee#4863]
Updating a feed with a new hash is almost instantaneous. 
However the content served by the hash may still be syncing and not available yest on the network. 
This leads to 404s and broken websites atleast for a certain amount of time.

Would be practical to check if the hash's content has been full synced or is available on the network by default. 
And only then update a feed hash. Maybe with a force option to override this check and update feed nevertheless.

**Comment by CanutoIX** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.

**Comment by martinconic** [comment]
Is the solution that fixed this issue [4734](https://github.com/ethersphere/bee/issues/4734) a valid solution for this one?
Previous fix: [4869](https://github.com/ethersphere/bee/pull/4869)
@istae

**Comment by istae** [comment]
This is true for almost all uploads that are deferred, use the tags API to monitor the sync status or use direct uploads to guarantee that before a reference is returned, all chunks have synced.
